syntax = "proto3";

service LesionSegmentation {
  rpc Segment (SegmentLesionRequest) returns (SegmentLesionResponse) {}
}

message SegmentLesionRequest {
  string extenstion = 1;
  int32 height = 2;
  int32 width = 3;
  bytes image = 4;
}

message SegmentLesionResponse {
  bytes segmentedImage = 1;
}
